docker-compose.dev.yml

```diff
@@ -23,12 +23,12 @@ services:
       - "./backend:/backend"
     command: yarn start:dev
     links:
-      - mongos
-      - shard-0
+      - mongodb-sharded
+      - mongodb-shard0
       - mongodb-cfg
     restart: always
 
-  mongos:
+  mongodb-sharded:
```

Comment:
name should be `mongos`

Reply:
got this 
![image](https://user-images.githubusercontent.com/42891316/99140138-85a0cd00-2671-11eb-832a-f65b12584b99.png)